functests/1_performance/cpu_management.go

```diff
@@ -41,6 +42,14 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 	var listReservedCPU []int
 	var reservedCPUSet cpuset.CPUSet
 
+	testutils.BeforeAll(func() {
+		isSingleNode, err := cluster.IsSingleNode()
+		Expect(err).ToNot(HaveOccurred())
+		if isSingleNode {
+			snoTimeoutMultiplier = 1
```

Comment:
tbh I pretty much followed what @mmirecki did, and I think they had a similar discussion on it https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/59#discussion_r572893473  
does it also apply here?

Reply:
good point, let me think about it.

Reply:
ok, the catch is different test do have indeed different timeouts.
So I'd start with something like
```

var RunningOnSingleNode bool

func computeTestTimeout(baseTimeout time.Duration) time.Duration {
    testTimeout := baseTimeout
    if RunningOnSingleNode {
        // let's explain the rationale here. We just observed that doubling the timeout fixes? there is any
        // law we discovered? do we need a lower bound for tests?
        testTimeout += baseTimeout
    }
    return testTimeout
}

testutils.BeforeAll(func() {
    RunningOnSingleNode, err := cluster.IsSingleNode()
    Expect(err).ToNot(HaveOccurred())
})
```

Use it like:

```
-                       }, 15*time.Minute, 15*time.Second).ShouldNot(HaveOccurred(), "Failed creating the performance profile")
+                       }, computeTestTimeout(15*time.Minute), 15*time.Second).ShouldNot(HaveOccurred(), "Failed creating the performance profile")
```

The idea is we should try hard to avoid to scatter around the code base low level details about SNO, multiplications and so forth.
If we observe that this approach doesn't scale - because there is not a common law, or we have too many different cases, or whatever, we will later revert inilining the computation in all the calling siters.
But let's at least try first!

Reply:
thanks for the comment
I changed it but haven't tested it on an SNO yet. I will update if doubling the timeout does the job.